imap_processing/spice/geometry.py

```diff
@@ -96,10 +97,11 @@ def imap_state(
     ----------
     et : np.ndarray or float
         Epoch time(s) [J2000 seconds] to get the IMAP state for.
-    ref_frame : SpiceFrame, optional
```

Comment:
I have no clue why sphinx was having problems with this `optional` here, which adheres to `numpydoc` syntax. I spent way too long trying to fix it but ended up just having to remove it.